recipes/cairo/meson/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1.2)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(cairo CONFIG REQUIRED)
+
+add_executable(${PROJECT_NAME} ../test_package/test_package.c)
```

Comment:
Just FYI, there as a new way for this https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt

less code :)